include/Graphs/SVFGNode.h

```diff
@@ -262,14 +264,25 @@ class MSSAPHISVFGNode : public MRSVFGNode
     typedef Map<u32_t,const MRVer*> OPVers;
 
 protected:
-    const MemSSA::MDEF* res;
+    const MRVer* resVer; 
     OPVers opVers;
 
 public:
     /// Constructor
-    MSSAPHISVFGNode(NodeID id, const MemSSA::MDEF* def,VFGNodeK k = MPhi): MRSVFGNode(id, k), res(def)
+    MSSAPHISVFGNode(NodeID id, const PointsTo pointsTo,VFGNodeK k = MPhi): MRSVFGNode(id, k)
```

Comment:
Unfortunately, this is a redundant method. Please remove this one and only keep `MSSAPHISVFGNode(NodeID id, const MRVer* resVer,VFGNodeK k = MPhi)`